### PR TITLE
Refuge translated to French

### PR DESCRIPTION
**Pull request by el00ruobuob**
As promises to @lh1008, this is the French translation of Refuge.  
This was really intersting to read through and dive into.  

Unfortunately, i couldn't fine any article reference in French for all those citation, just one for the European forfeiture law (from our French senate).

### CODE REVIEW

**Comment by erciccione** [comment]
Maybe @assilyas could help with the review?

**Comment by el00ruobuob** [comment]
Thank you @assylias!  
I will correct those soon.

File: monero-outreach-docs/translations/fr/storyboard/refuge_fr.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+## Confiscation civile aux États-Unis
+
+*“L'idée de s'en prendre aux populations via leur propriété est une vieille histoire. Ces théories sont assez anciennes. La prédominance de cette pratique est par contre relativement récente.”* - Daniel C. Richman (1999) 
+
+Pour faire simple, [la confiscation civile désigne le processus juridique](https://www.law.cornell.edu/wex/civil_forfeiture) par lequel les autorités chargées de l'application de la loi "saisissent" des biens (argent, immeubles et autres biens corporels) de personnes soupçonnées d'avoir participé à une activité criminelle. *Cela semble louable, n'est-ce pas ?* *Après tout, le monde ne serait-il pas meilleur si nous pouvions empêcher les actes criminels avant même qu'ils ne se produisent ?* Bien que cela le semble peut-être solide en théorie, ce processus juridique imprécis et intrusif a malheureusement donné lieu à d'innombrables problèmes et à des excès flagrants de la part du gouvernement.
```
[the remark this answers is not in the document]
Comment (el00ruobuob) [reply monero-outreach-docs/translations/fr/storyboard/refuge_fr.md]: ok

File: monero-outreach-docs/translations/fr/storyboard/refuge_fr.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+Puisant ses racines dans le droit maritime britannique et remontant aux XVIIe siècle, la confiscation civile est devenue une industrie gouvernementale en milliards de dollars au cours des 30 dernières années, entraînant une tendance à la hausse et agressive vers les abus, passant d'environ 93,7 millions de dollars en biens et argent saisis en 1986 [à plus de 2,5 milliards de dollars en 2010](https://www.wsj.com/news/articles/SB10001424053111903480904576512253265073870).
```
[the remark this answers is not in the document]
Comment (el00ruobuob) [reply monero-outreach-docs/translations/fr/storyboard/refuge_fr.md]: ok

File: monero-outreach-docs/translations/fr/storyboard/refuge_fr.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+Et bien qu'il puisse y avoir du bien fondé, un motif raisonnable ou des preuves prépondérantes (les États américains diffèrent dans leurs règles de preuve respectives en matière de lois de confiscation) en faveur de la saisie de certains biens, des estimations établissent [à environ 85 % au fil des ans](http://www.sfgate.com/news/nation-world/article/Asset-forfeiture-both-an-effective-tool-4546043.php) les cas non fondés et fantaisistes de confiscations civiles. Rien qu'en 2010, on estime qu'il y a eu 11 000 cas de confiscation non criminelle aux États-Unis, dont seulement un peu plus du dixième ont été contestés devant une cour fédérale. Selon une estimation de l'auteur de SFGate, Stewart M. Powell, le pourcentage des biens saisis par le gouvernement fédéral qui sont restitués aux propriétaires serait [d'un piètre 1%](https://www.sfgate.com/news/nation-world/article/Asset-forfeiture-both-an-effective-tool-4546043.php).
```
[the remark this answers is not in the document]
Comment (el00ruobuob) [reply monero-outreach-docs/translations/fr/storyboard/refuge_fr.md]: ok

File: monero-outreach-docs/translations/fr/storyboard/refuge_fr.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
..., il existe un programme de « [Partage équitable](https://www.justice.gov/criminal-mlars/equitable-sharing-program) », un programme dans le cadre duquel le produit de la liquidation des biens saisis et confisqués est réparti entre les autorités chargées de l'application de la loi au niveau fédéral et à celui des États. En 2010, plus de [500 millions de dollars ont été partagés entre les agences et leurs représentants](https://www.wsj.com/articles/SB10001424053111903596904576514583716653122) via le programme de partage équitable, pour plus de 5 milliards de dollars distribué depuis l'inauguration du programme en 1984 ; une [caisse noire](http://www.linternaute.fr/expression/langue-francaise/15510/caisse-noire/) pour les oints. Si vous pensez que ces fonds ne servent qu'à faire progresser les tâches policières, vous vous trompez complètement ; les agents des forces de l'ordre et leurs représentants ont également utilisé ces fonds « équitablement partagés » pour [subventionner des fêtes, des dîners et des dépenses personnelles onéreuses](https://harvardlawreview.org/2018/06/how-crime-pays-the-unconstitutionality-of-modern-civil-asset-forfeiture-as-a-tool-of-criminal-law-enforcement/).
```
[the remark this answers is not in the document]
Comment (el00ruobuob) [reply monero-outreach-docs/translations/fr/storyboard/refuge_fr.md]: ok

File: monero-outreach-docs/translations/fr/storyboard/refuge_fr.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+
+Gardez à l'esprit qu'il ne s'agit là que d'une fraction [des milliards qui sont saisis chaque année](https://harvardlawreview.org/2018/06/how-crime-pays-the-unconstitutionality-of-modern-civil-asset-forfeiture-as-a-tool-of-criminal-law-enforcement/) de manière illégitime. 
+  
+## Comment Monero peut aider les populations et les préserver des lois inéquitables en matière de confiscation civile
+
+Monero, une monnaie numérique sécurisée, rapide et, surtout, confidentielle, peut profiter à ceux qui autrement pourraient être victimes de violations constitutionnelles et d'interprétations vagues et pernicieuses de la loi. 
+
+En ce qui concerne les lois inéquitables des États-Unis (et d'une [minorité d'autres juridictions](https://www.senat.fr/ue/pac/E7171.html)) en matière de confiscation civile, Monero se permet spécifiquement d'empêcher ce qui suit :
```
[the remark this answers is not in the document]
Comment (el00ruobuob) [reply monero-outreach-docs/translations/fr/storyboard/refuge_fr.md]: ok

File: monero-outreach-docs/translations/fr/storyboard/refuge_fr.md [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+## Comment Monero peut aider les populations et les préserver des lois inéquitables en matière de confiscation civile
+
+Monero, une monnaie numérique sécurisée, rapide et, surtout, confidentielle, peut profiter à ceux qui autrement pourraient être victimes de violations constitutionnelles et d'interprétations vagues et pernicieuses de la loi. 
+
+En ce qui concerne les lois inéquitables des États-Unis (et d'une [minorité d'autres juridictions](https://www.senat.fr/ue/pac/E7171.html)) en matière de confiscation civile, Monero se permet spécifiquement d'empêcher ce qui suit : 
+
+1. Le comportement prédateur du gouvernement et de la police ;
+2. Les sanctions disproportionnées et arbitraires.
+
+Voyons rapidement comment Monero peut permettre de les empêcher.
```
[the remark this answers is not in the document]
Comment (el00ruobuob) [reply monero-outreach-docs/translations/fr/storyboard/refuge_fr.md]: ok

File: monero-outreach-docs/translations/fr/storyboard/refuge_fr.md [hunk 7]
```diff
@@ -0,0 +1,95 @@
...
+
+*Mais que se passe-t-il si les actifs et la richesse d'une personne sont détenus sous forme numérique ?*
+
+S'il n'y a pas de bien matériel visible d'un agent ou d'un représentant, la probabilité de perquisition et de saisie diminue considérablement. 
+
+Monero et les autres cryptomonnaies préservant la confidentialité arrivent. Bien que la plupart des cryptomonnaies P2P et des cryptomonnaies pseudo-anonymes constituent un pas en avant vers l'émancipation des lois actuelles en matière de confiscation civile, qu'elles soient prédatrices ou au bras long, elles ne sont tout de même pas protégées, ayant fait l'objet d'une certaine surveillance et d'un certain suivi par [des sociétés privées et des organismes gouvernementaux grâce aux sorties de transactions non dépensées](https://bitcoinist.com/yes-your-bitcoin-transactions-can-be-tracked-and-here-are-the-companies-that-are-doing-it/).
```
[the remark this answers is not in the document]
Comment (el00ruobuob) [reply monero-outreach-docs/translations/fr/storyboard/refuge_fr.md]: ok

File: monero-outreach-docs/translations/fr/storyboard/refuge_fr.md [hunk 8]
```diff
@@ -0,0 +1,95 @@
...
+
+Monero, en revanche, opère en utilisant des différences substantielles (par rapport à Bitcoin) en termes d'offuscation de la chaîne de blocs, grâce au [protocole CryptoNote](https://cryptonote.org/), selon lequel des signatures de cercle masquent l'entrée d'un expéditeur parmi une multitude d'autres entrées possibles, ce qui rend plus difficile de lier des transactions aux suivantes. [Malte Möser, de l'Université de Princeton, a fait remarquer qu'environ 62 % des transactions Monero](https://www.technologyreview.com/s/610807/sitting-with-the-cyber-sleuths-who-track-cryptocurrency-criminals/) étaient vulnérables à une analyse de démêlage (en avril 2018), non par manque de protocoles adaptés, mais plutôt par erreur humaine et pratiques de confidentialité peu rigoureuses.
```
[the remark this answers is not in the document]
Comment (el00ruobuob) [reply monero-outreach-docs/translations/fr/storyboard/refuge_fr.md]: ok

File: monero-outreach-docs/translations/fr/storyboard/refuge_fr.md [hunk 9]
```diff
@@ -0,0 +1,95 @@
...
+
+Grâce aux signatures de cercle de CryptoNote, en cas de [saisie d'un ordinateur (ou 130) par les forces de l'ordre et leurs représentants](https://www.heritage.org/economic-and-property-rights/commentary/civil-asset-forfeiture-has-created-serious-problems), ils ne réussiront jamais à déchiffrer les origines des fonds, car tous les membres d'un groupe de signataires sont des signataires potentiels. En particulier, la fongibilité de Monero élimine tout « [élément d'identification unique qui pourrait lier les transactions d'une seule unité XMR dans le temps](https://www.monerooutreach.org/preserving-fundamental-rights.php) ». ADe plus, aucun compromis mesurable n'est consenti sur la confidentialité des tiers en raison de l'absence de lien véritable entre les parties. Grâce à Monero, les identités de l'expéditeur et du destinataire sont protégées cryptographiquement, ainsi que le montant de leurs transactions.
```
[the remark this answers is not in the document]
Comment (el00ruobuob) [reply monero-outreach-docs/translations/fr/storyboard/refuge_fr.md]: ok

**Comment by el00ruobuob** [comment]
Corrections applied.

**Comment by el00ruobuob** [comment]
Ready to merge @erciccione & @lh1008

**Pull request merged**